src/wymeditor/editor/gecko.js

```diff
@@ -146,7 +146,8 @@ WYMeditor.WymClassGecko.prototype._designModeQuirks = function () {
     try {
         wym._doc.execCommand("styleWithCSS", '', false);
         wym._doc.execCommand("enableObjectResizing", false, false);
-        wym._doc.execCommand("enableInlineTableEditing", false, false);
```

Comment:
Brah, WYMeditor is such an old editor. Why don't you use some more modern alternative?